metricbeat/module/system/process/process.go

```diff
@@ -0,0 +1,114 @@
+/*
+
+An example event looks as following:
+
+
+    {
+      "@timestamp": "2016-04-26T19:24:19.108Z",
+      "beat": {
+        "hostname": "ruflin",
+        "name": "ruflin"
+      },
+      "metricset": "process",
+      "module": "system",
+      "rtt": 20982,
+      "system-process": {
```

Comment:
@tsg @simianhacker I went now with nested documents for the processes list (same for filesystem): https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html Nested queries, aggregation and scripts should help us to make sense of it.